libs/deps/mas/plans-modal.js

```diff
@@ -4232,6 +4232,7 @@ var setCustomTemplateLiteralTag = (e16) => {
 
 // ../node_modules/@spectrum-web-components/icons-ui/src/icons/Cross200.js
 var Cross200Icon = ({ width: t13 = 24, height: e16 = 24, title: r9 = "Cross200" } = {}) => tag`<svg
+    aria-hidden="true"
```

Comment:
All of the SVGs in this file already have the attribute, these changes shouldn't be needed.

Reply:
There are some that did not have, but this one did. I removed the duplicate 👍